### ISSUE TITLE
Problem with import

### ISSUE THREAD

**Issue by leschek** [danny0838/webscrapbook#127]
I followed the FAQ find here: https://github.com/danny0838/webscrapbook/wiki/Import, but when I try to import tree/map.html I get this error:

> Error on importing options: JSON.parse: unexpected character at line 1 column 1 of the JSON data

I'm using WebScrapBook 0.56.3 on FF 70.0.1 (64-bit)

**Comment by danny0838** [comment]
There is no step `import tree/map.html`. Can you describe what you have actually done? Or provide a video showing how you attempted to perform the importing?

**Comment by leschek** [comment]
Sorry for the late reply and probably for my misunderstanding how import works.
Basically I followed all the steps on import FAQ page as linked in first post of this issue. I opened and checked the tree/map.html and it is working.
I also read this sentence on the same page:

> After that the original ScrapBook X data tree will be available using the generated tree/map.html or tree/frame.html

and I thought that I just load it with the "import" button to load it to webscrapbook (this is where I get the error mentioned in first post) and it will be working as old Scrapbook. 

Once more sorry for my misunderstanding and for wasting your time.
